From: Bernhard M. Wiedemann <bmwiedemann suse de>
Set BUILD_TIME to a constant value
so that TIME values in .beam files do not differ across builds

Index: rebar-2.6.4+git20160922.576e121/bootstrap
===================================================================
--- rebar-2.6.4+git20160922.576e121.orig/bootstrap
+++ rebar-2.6.4+git20160922.576e121/bootstrap
@@ -56,7 +56,6 @@ main(Args) ->
                     [{outdir, "ebin"}, {i, "include"},
                      DebugFlag,
                      NamespacedTypes,
-                     {d, 'BUILD_TIME', Built},
                      {d, 'VCS_INFO', VcsInfo},
                      {d, 'OTP_INFO', OtpInfo}]) of
         up_to_date ->
